public/modules/custom/grants_profile/src/ProfileConnector.php

```diff
@@ -44,7 +44,11 @@ class ProfileConnector {
    * @param \Drupal\helfi_yjdh\YjdhClient $yjdhClient
    *   Access to yjdh data.
    */
-  public function __construct(HelsinkiProfiiliUserData $helsinkiProfile, MunicipalityService $municipalityService, YjdhClient $yjdhClient) {
+  public function __construct(
+    HelsinkiProfiiliUserData $helsinkiProfile,
+    MunicipalityService $municipalityService,
```

Comment:
These should use constructor promotion (https://stitcher.io/blog/constructor-promotion-in-php-8)